utils/src/ooinstall/cli_installer.py

```diff
@@ -120,24 +120,26 @@ def collect_hosts(oo_cfg, existing_env=False, masters_set=False, print_summary=T
     click.echo(message)
 
     hosts = []
+    roles = set(['master', 'node', 'storage'])
```

Comment:
As long as the additional roles get added as needed (or when pulled in from config), then no objections.  I made my comments as I read top-down.